src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import React from 'react';
+import ActivityError from '../common/activity-error';
+import ActivityThread from '../activity-feed/ActivityThread';
+import AnnotationActivity from '../annotations';
+import API from '../../../../api/APIFactory';
+import LoadingIndicator from '../../../../components/loading-indicator/LoadingIndicator';
+import useAnnotationAPI from './useAnnotationAPI';
+
+import type { BoxItemPermission, SelectorItems } from '../../../../common/types/core';
+import type { ElementOrigin, ElementsXhrError } from '../../../../common/types/api';
+
+import './AnnotationThreadContent.scss';
+
+type Props = {
+    annotationId: string,
+    api: API,
+    fileId: string,
+    filePermissions: BoxItemPermission,
+    onError: (error: ElementsXhrError | Error, code: string, contextInfo?: Object, origin?: ElementOrigin) => void,
+};
+
+const AnnotationThreadContent = ({ annotationId, api, fileId, filePermissions, onError }: Props) => {
+    const [mentionSelectorContacts, setMentionSelectorContacts] = React.useState([]);
+
+    const { annotation, isLoading, error, handleEdit, handleStatusChange, handleDelete } = useAnnotationAPI({
+        api,
+        fileId,
+        annotationId,
+        filePermissions,
+        errorCallback: onError,
+    });
+
+    /**
+     * File @mention contacts fetch success callback
+     *
+     * @private
+     * @param {BoxItemCollection} collaborators - Collaborators response data
+     * @return {void}
+     */
+    const getMentionContactsSuccessCallback = ({ entries }: { entries: SelectorItems<> }): void => {
+        setMentionSelectorContacts(entries);
+    };
+
+    const getMentions = (searchStr: string) =>
+        api
+            .getFileCollaboratorsAPI(false)
+            .getCollaboratorsWithQuery(fileId, getMentionContactsSuccessCallback, onError, searchStr);
+
+    const getAvatarUrl = async (userId: string): Promise<?string> => {
+        return api.getUsersAPI(false).getAvatarUrlWithAccessToken(userId, fileId);
```

Comment:
Above, we're using implicit return, but here we're using explicit return. We also declare the return type in some places but not others. Can we make it consistent?